python/pyspark/context.py

```diff
@@ -860,6 +860,19 @@ def addPyFile(self, path):
             import importlib
             importlib.invalidate_caches()
 
+    def addJar(self, path, addToCurrentClassLoader=False):
```

Comment:
We should mention that adding a jar to the current class loader is a developer API and may change.